parser/remote/hiveql/pom.xml

```diff
@@ -13,15 +13,19 @@
   
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.10</version>
-      <scope>test</scope>
+      <groupId>com.google.guava</groupId>
```

Comment:
All review in calcite Pom May apply here too

Reply:
I see. Will do.